### ISSUE TITLE
Reason for higher performance of ComiRec-SA

### ISSUE THREAD

**Issue by xyq7** [ShiningCosmos/pytorch_ComiRec#3]
Hi! Thank you for your nice reproduction. 
I found that we can get a higher performance of ComiRec-SA than the original repo. Do you think it is because of the loss function difference? Or are there any other differences?

Thanks!

**Comment by ShiningCosmos** [comment]
I'm not sure why.
The loss function may be one of the reasons. In addition, I use different parameter initialization methods (torch.nn.init.kaiming_normal_).
In my understanding, Tensorflow and Pytorch have some differences:
(1) Some network layers are implemented differently.
(2) The default parameters of the network layer are different, such as the initialization method.
(3) The random process is different.
